experimental/segmenter/src/grapheme.rs

```diff
@@ -173,7 +173,7 @@ impl GraphemeClusterSegmenter {
 
     /// Create a grapheme cluster break iterator from grapheme cluster rule payload.
     ///
-    /// If the string is empty, no breakpoints are returned.
+    /// There are always breakpoints at 0 and the string length, or only at 0 for the empty string.
```

Comment:
I don't think `, or only at 0 for the empty string` is necessary, as we don't return duplicate indices.

Reply:
I agree but don't feel strongly.

Reply:
I felt it was important to call out because it is the only case where we return only 1 entry